Assets/Scripts/CollisionImpulseTracker.cs

```diff
@@ -5,21 +5,20 @@
 /// </summary>
 public class CollisionImpulseTracker : MonoBehaviour
 {
-	[Tooltip("Tracks the cumulative impulse magnitude from collisions this frame.")]
```

Comment:
Why remove this tooltip?

Reply:
I dont think it's needed as there is a class summary that says the same thing. There is no tooltip statements throughout the project and I dont think it makes a difference here.